Yoast/wordpress-seo
# Replace 'SEO Available' with more descriptive text

## Summary

This PR can be summarized in the following changelog entry:

* Replaces 'SEO Available' with 'N/A - Focus Keyphrase not set' to provide more descriptive text in the publish box when SEO rank can't be generated.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Enable the Classic Editor.
* Go to the edit screen for a post.
* Remove the Focus Keyphrase in the Yoast Metabox, if one exists.
* See the "SEO: N/A - Focus Keyphrase not set" text in the publish box.

## UI changes

## Documentation

## Quality assurance


Fixes #11882
